src/main/java/com/fasterxml/jackson/annotation/JsonFormat.java

```diff
@@ -22,6 +22,11 @@
  * <li>{@link java.util.Date}: Shape can  be {@link Shape#STRING} or {@link Shape#NUMBER};
```

Comment:
Not part of change here, but should probably add `java.util.Calendar` along with `java.util.Date`.